packages/paste-website/src/component-examples/ListboxPrimitiveExamples.ts

```diff
@@ -0,0 +1,250 @@
+        <ListboxPrimitiveItem
+          key={item}
+          {...listbox}
+          selected={selected === item}
+          onSelect={() => {
+            setSelected(item);
+          }}
+        >
+          {item}
+        </ListboxPrimitiveItem>
+      ))}
+    </ListboxPrimitive>
+  )
+};
+
+render(
+  <HorizontalExample />
+)
+`.trim();
+
+export const groupsExample = `
+const GroupsExample = () => {
+  const [items] = React.useState(['Item 1', 'Item 2', 'Item 3']);
+  const [selected, setSelected] = React.useState();
+  const renderItem = (item, index) => (
+    <ListboxPrimitiveItem
+      key={item}
+      {...listbox}
+      selected={selected === item}
+      onSelect={() => {
+        setSelected(item);
+      }}
+    >
+      {item}
+    </ListboxPrimitiveItem>
+  );
+  const listbox = useListboxPrimitiveState();
+  return (
+    <ListboxPrimitive {...listbox} aria-label="My listbox">
+      <ListboxPrimitiveGroup aria-labelledby="group-1">
+        <Stack orientation="vertical" spacing="space0">
+          <span id="group-1">Even</span>
+          {items.filter((item, index) => (index + 1) % 2 === 0).map(renderItem)}
+        </Stack>
+      </ListboxPrimitiveGroup>
+      <ListboxPrimitiveGroup aria-labelledby="group-2">
+        <Stack orientation="vertical" spacing="space0">
+          <span id="group-2">Odd</span>
+          {items.filter((item, index) => (index + 1) % 2 === 1).map(renderItem)}
+        </Stack>
+      </ListboxPrimitiveGroup>
+    </ListboxPrimitive>
+  )
+};
+
+render(
+  <GroupsExample />
+)
+`.trim();
+
+export const multipleExample = `
+const MultipleExample = () => {
+  const [items] = React.useState(['Item 1', 'Item 2', 'Item 3']);
+  const [selectedSet, updateSelectedSet] = React.useState(new Set());
+  const listbox = useListboxPrimitiveState({orientation: 'horizontal'});
+  return (
+    <ListboxPrimitive {...listbox} aria-label="My listbox" variant="multiple">
+      {items.map((item) => (
+        <ListboxPrimitiveItem
+          key={item}
+          {...listbox}
+          selected={selectedSet.has(item)}
+          onSelect={() => {
+            const newSelectedSet = new Set(selectedSet);
+            if (newSelectedSet.has(item)) {
+              newSelectedSet.delete(item);
+            } else {
+              newSelectedSet.add(item);
+            }
+            updateSelectedSet(newSelectedSet);
+          }}
+        >
+          {item}
+        </ListboxPrimitiveItem>
+      ))}
+    </ListboxPrimitive>
+  )
+};
+
+render(
+  <MultipleExample />
+)
+`.trim();
+
+export const dualExample = `
+const DualExample = () => {
+  const [components, updateComponents] = React.useState(['Alert', 'Anchor', 'Button', 'Card', 'Heading', 'List']);
+  const [favs, updateFavs] = React.useState(['Modal']);
+  const [selectedComps, updateSelectedComps] = React.useState(new Set());
+  const [selectedFavs, updateSelectedFavs] = React.useState(new Set());
+  const listbox = useListboxPrimitiveState();
```

Comment:
Issue: can you separate the lists into two separate listboxes? currently, if I press down arrow on the last item in the first list, I go to the first item in the second list.